test/irmin-pack/bench.ml

```diff
@@ -39,12 +38,12 @@ let index root =
       let s = file (Filename.concat root filename) in
       aux (acc + s) (i + 1)
   in
-  aux 0 0 / 1024 / 1024
-
-let log root = file (Filename.concat root "store.log")
```

Comment:
Here we were previously reading the size of a log file that (I believe) never exists.